contributors/guide/scalability-good-practices.md

```diff
@@ -70,20 +66,18 @@ func (s Scheduler) DoSchedule(podsChan chan v1.Pod) {
  
-As there are no constant references in go this is the only option of passing objects without copying them (except creating read-only interfaces for all types, but this is infeasible). Note that it is, and should be, scary to pass a pointer to your object to strangers. Before you do so please make sure that the code you're passing the pointer to won't modify the object. Races are bad as well. Note that all `Informers` (see next paragraph) caches are expected to be immutable.
+As there are no constant references in go, this is the only option for passing objects without copying them (except creating read-only interfaces for all types, but that is not feasible). Note that it is (and should be) scary to pass a pointer to your object to strangers. Before you do so, make sure the code to which you are passing the pointer will not modify the object. Races are bad as well. Note that all `Informers` (see next paragraph) caches are expected to be immutable.
  
-I could go on, and on, but I think the point is clear - when writing code that will be executed often you need to think about memory management. From time to time every one of us wants to be lazy and not think about it at all, but this pretty much always bites us back when we look at performance. General rules are: using heap is very expensive (garbage collection) - you should avoid unnecessary heap operations altogether, copying objects a lot is noticeable - it should be minimized. Learn how go manages memory. This is especially important in components running on the master. Otherwise we may end up in the situation when API server is starved on CPU and can't respond quickly to requests.
+We could go on and on, but the point is clear -- when writing code that will be executed often, you need to think about memory management. From time to time every one of us wants to be lazy and not think about it at all, but this always bites us when we look at performance. General rules are: using heap is very expensive (garbage collection) - you should avoid unnecessary heap operations altogether, repeatedly copying objects is noticeable - it should be minimized. Learn how go manages memory. This is especially important in components running on the master. Otherwise we may end up in the situation where the API server is starved on CPU and cannot respond quickly to requests.
```

Comment:
- rephrase: remove "lazy" and "bites us". Let's not use slang with an international audience, and certainly not phrasing with negative connotation.
- Turn everything after "General rules" into a bullet list.
- "Go" or "Golang"
- "Control plane" instead of "master".

Reply:
Done.